visualization/.eslintrc.js

```diff
@@ -37,7 +37,6 @@ module.exports = {
 		"@typescript-eslint/no-unnecessary-type-arguments": "error",
 		"@typescript-eslint/no-unused-expressions": "error",
 		"@typescript-eslint/no-unused-vars": ["error"],
-		"@typescript-eslint/prefer-nullish-coalescing": "error",
```

Comment:
Any reason to remove this?

Reply:
Great question I forgot to mention in the PR! Yes, it requires "strictNullChecks" in tsconfig, which we don't have as stated in [its docs](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md). It broke my IDE :/ Adding "strictNullCheck" in the future is something I would love, but out of scope here, I think.

> :::caution This rule will not work as expected if [strictNullChecks](https://www.typescriptlang.org/tsconfig#strictNullChecks) is not enabled. :::

Reply:
If I am not mistaken, the rule just does not work but it does not cause any false positives either, no?

Reply:
My vscode blamed a lot of import destructions like `import { xy } from "xyz"` after a fresh install, that it could be null... 💁‍♂️